internal/controller/pod_webhook.go

```diff
@@ -58,7 +58,7 @@ func (a *PodAnnotator) Handle(ctx context.Context, req admission.Request) admiss
 	}
 
 	// Add scheduling
-	schedulingGateName := "org.instaslice/accelarator"
```

Comment:
`"org.instaslice/accelarator"` is defined in multiple files, maybe a good idea to centralize.

Reply:
I defined this in internal/controller/constants.go.